ts/packages/cli/src/base.ts

```diff
@@ -76,6 +76,19 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
     ux.action.stop(` ${mark} ${chalk.dim(text ?? defaultText)}`);
   }
 
+  getCacheStoreInstanceByStoreFlag(storeFlag?: string) {
+    if (!storeFlag) {
+      throw new Error('--store flag is missing');
+    }
+
+    const storeCache = this.config.cli.data.stores?.[storeFlag]?.cache;
+    if (!storeCache) {
+      return false;
+    }
+    const database = typeof storeCache === 'boolean' ? path.join(this.config.cacheDir, 'cache.db') : storeCache;
```

Comment:
`cache.db` is incorrect, the [issue](https://github.com/configu/configu/issues/370#issuecomment-1980847308) says:
> * When `true`, use the default path `$cacheDir/$storeName.sqlite`.

Reply:
You should always update the code before reviewing it. This is irrelevant

Reply:
Irrelevant

Reply:
`config.sqlite`

Reply:
done